src/hawkmoth/doccursor.py

```diff
@@ -116,7 +124,9 @@ def quals(self):
                              CursorKind.DESTRUCTOR,
                              CursorKind.CXX_METHOD,
                              CursorKind.FUNCTION_TEMPLATE]:
-            return ' '.join(self._get_method_quals()[1])
+            def gen(cursor):
```

Comment:
I forgot I was taking the name of the object as the cache key. `gen` is a bit too generic then... I'll fix this up too.

Reply:
Oh, it's actually much worse! I key on the generator object id / name, but they may be called with different arguments... Fixing.